src/Core/Framework/App/Command/InstallAppCommand.php

```diff
@@ -86,10 +86,12 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             }
 
             try {
-                // in the future: if it was forced then it counts as not accepted
                 $this->appLifecycle->install(
                     $manifest,
-                    new AppInstallParameters(activate: $input->getOption('activate')),
+                    new AppInstallParameters(
+                        activate: $input->getOption('activate'),
+                        acceptPermissions: $input->getOption('force') === false
```

Comment:
@Gaitholabi and me are not sure but believe this should be:

```suggestion
                        acceptPermissions: $input->getOption('force') === true
```

So if the `--force` flag is passed, we `acceptPermissions: true`, if not we don't.

Reply:
I agree, if force is set we don't explicitly ask for all the permissions, we just grant them, therefore I also would assume that all permissions are accepted by default with the --force flag

Reply:
the comment: `// in the future: if it was forced then it counts as not accepted` is pretty explicit about it, hence the confusion 🤔

Reply:
@fschmtt @keulinho I'm leaning to say that both conditionals are wrong, and this should simply be set to 'true'.

In the context of the command, when the execution reaches this point it's either that: `--force` is set, or the permissions were already printed, and the user acknowledged.